docs/components/flags/compact.txt-e

```diff
@@ -0,0 +1,114 @@
+usage: thanos compact [<flags>]
+
```

Comment:
We don't need to commit those ): Should be git ignored, but I guess you "escaped" gitignore with `-e` file suffix.